htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php

```diff
@@ -942,6 +942,8 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			$societeforaction->fetch($object->socid);
 		} elseif (isset($object->fk_soc) && $object->fk_soc > 0) {
 			$societeforaction->fetch($object->fk_soc);
+		} elseif (isset($object->thirdparty->id) && $object->thirdparty->id > 0) {
```

Comment:
If $object->thirdparty is not defined, it will crash on php8. Can you also add a test
if (isset($object->thirdparty->id) ?